backend/src/export/export.service.ts

```
@@ -0,0 +1,329 @@
+          if (hasAudio) {
+            const audioReadme = `Audio Files for ${episodeTitle}\n\nThis CBZ archive includes audiobook narration for each page.\nAudio files are located in the 'audio/' directory.\n\nFile naming: page_XXX.mp3 corresponds to page_XXX.png\n`;
+            archive.append(audioReadme, { name: 'audio/README.txt' });
+          }
+        }
+
+        // Finalize the archive
+        archive.finalize();
+      } catch (error) {
+        reject(error);
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)